apistructs/issue_stream_locale.go

```diff
@@ -72,24 +72,33 @@ var IssueTemplate = map[string]map[IssueStreamType]string{
 }
 
 const (
-	ChildrenInProgress    = "childrenInProgress"
-	MrCreated             = "mrCreated"
-	IterationChanged      = "iterationChanged"
-	PlanFinishedAtChanged = "planFinishedAtChanged"
+	ChildrenInProgress     = "childrenInProgress"
+	MrCreated              = "mrCreated"
+	IterationChanged       = "iterationChanged"
+	PlanFinishedAtChanged  = "planFinishedAtChanged"
+	ChildrenPlanUpdated    = "childrenPlanUpdated"
+	ParentLabelsChanged    = "parentLabelsChanged"
+	ParentIterationChanged = "parentIterationChanged"
```

Comment:
it seems more and more case came out, shall we use i18n pkg instead of hard-coded template?

Reply:
Changed on newly added "reasonDetail", add todo for legacy issue stream templates.